.github/workflows/performance-tests.yml

```diff
@@ -36,12 +36,12 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # https://github.com/microsoft/vscode-ripgrep/issues/9
 
       - name: Performance (browser)
-        uses: GabrielBB/xvfb-action@v1
+        uses: coactions/setup-xvfb@v1
         with:
           run: yarn performance:startup:browser
 
       - name: Performance (Electron)
-        uses: GabrielBB/xvfb-action@v1
+        uses: coactions/setup-xvfb@v1
```

Comment:
I suggest we give it a try without this action - as I commented on the PR, it would seems `ubuntu-latest` comes with `xvfb` pre-installed.

Reply:
I think these are the same errors today in repo theia-playwright-template, that went away after pinning Python to 3.11:
https://github.com/eclipse-theia/theia/actions/runs/8286508515/job/22676662833

Reply:
I updated the PR to use `xvfb-run` directly. Thank you for pointing this out!